pages/basePage.ts

```diff
@@ -0,0 +1,34 @@
+import { Page, Locator } from '@playwright/test';
+
+export class MainPage {
+  private page: Page;
+
+  private elementsText = "Elements";
+  private formsText = "Forms";
+  private alertsText = "Alerts, Frame & Windows";
+  private widgetsText = "Widgets";
+  private interactionsText = "Interactions";
+  private bookStoreText = "Book Store Application";
+
+  constructor(page: Page) {
+    this.page = page;
```

Comment:
you can also declare variables here:
```suggestion
    this.page = page;
    this.elementsCategory = this.page.getByText(this.elementsText);
```